src/scripts/main.js

```diff
@@ -1 +1,74 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.documentElement.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  // eslint-disable-next-line prefer-promise-reject-errors
+  window.setTimeout(() => reject('First promise was rejected'), 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.documentElement.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  document.documentElement.addEventListener('contextmenu', () => {
+    resolve('Second promise was resolved');
```

Comment:
The second promise resolves on both 'click' and 'contextmenu' events. This means it will resolve twice if both events occur, which might not be the intended behavior. Consider whether this logic meets your requirements.